app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,70 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_food, only: %i[show edit update destroy]
+
+  # GET /foods or /foods.json
+  def index
```

Comment:
A user should be able to see all food items created by other users as well. I am able to see only food items that are created by the currently logged-in user on the food index page.